src/adversarial/neurify.jl

```diff
@@ -26,29 +26,45 @@ Sound but not complete.
 """
 
 @with_kw struct Neurify
-    max_iter::Int64     = 1000
+    max_iter::Int64     = 10
     tree_search::Symbol = :DFS # only :DFS/:BFS allowed? If so, we should assert this.
+    model = Nothing()
+    optimizer = GLPK.Optimizer
+    splitted = Set() # To prevent infinity loop. Not the best solution, need to be modified in the future.
 end
 
 struct SymbolicInterval{F<:AbstractPolytope}
     Low::Matrix{Float64}
     Up::Matrix{Float64}
     interval::F
+    model
```

Comment:
I'm not seeing this new field used anywhere. Do we need it?

Reply:
Not anymore. Will remove.